test/Storage/MessageTest.php

```diff
@@ -415,6 +416,35 @@ public function testMultivalueToHeader()
         $this->assertEquals('nicpoń', $addressList->get('bar@example.pl')->getName());
     }
 
+    /**
+     * Test case for creating mail with Mail\Message containing \n\n,
+     * serializing it to text and loading with Storage\Message.
+     *
+     * Testcase showing that if mail is created with text body
+     * that is \n separated, loading such mail with Storage\Message
+     * causes parse error because it goes into mail body to find headers.
+     */
+    public function testDogFood() {
+        $text = "Hello, bödi tekst\n\nBye";
+
+        $part = new Mime\Part($text);
+        $part->type = 'text/plain';
+        $part->charset = 'UTF-8';
+
+        $mime = new Mime\Message();
+        $mime->addPart($part);
```

Comment:
None of the preceding 7 lines are ever used within the test. Is your setup missing something (attaching a MIME part to the message)? Or can the lines be removed safely?

Reply:
i think they can be removed as the reproducer worked. otherwise i wouldn't had pushed it this way.